sys/net/crosslayer/ng_pktbuf/ng_pktbuf.c

```diff
@@ -258,6 +258,16 @@ static ng_pktsnip_t *_pktbuf_add_unsafe(ng_pktsnip_t *pkt, void *data,
             return NULL;
         }
 
+        DEBUG("pktbuf: Adding chunk to %p ", pkt->data);
+
+        if (!_pktbuf_internal_add_pkt(pkt->data)) {
```

Comment:
> On the other hand this now checks pkt->data before it is incremented. On purpose?

Makes no different. The pointer only needs to be an address in the chunk that is representing the packet internally.